compute-roadmap.md

```
@@ -0,0 +1,522 @@
+
+**5. `internal/api/setup/router.go`** — wire endpoint
+```go
+imageGroup.POST("/import", httputil.Permission(svcs.RBAC, domain.PermissionImageCreate), handlers.Image.ImportImage)
+```
+
+**6. Tests**
+- Unit: `image_unit_test.go` — add `ImportImage` subtests
+- E2E: `tests/compute_e2e_test.go` if applicable
+
+### Files
+| Action | File |
```

markdownlint-cli2 (0.22.1)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)